mappings/sweet-envo_cryosphere.sssom

```diff
@@ -0,0 +1,175 @@
+obo:ENVO_00000132	ice sheet	sweet:realmCryo/IceSheet	IceSheet	skos:closeMatch	HumanCurated	"A candidate for exact match if SWEET adds ENVO's definition to SWEET's term"
+obo:ENVO_00000380	ice shelf	sweet:realmCryo/IceShelf	IceShelf	skos:closeMatch	HumanCurated	"A candidate for exact match if SWEET adds ENVO's definition to SWEET's term"
+obo:ENVO_01001542	ice surface layer	sweet:realmCryo/IceSurface	IceSurface	skos:relatedMatch	HumanCurated	"Related as the ENVO class is more generic; the SWEET class is specifically a planetary bounding layer."
+obo:ENVO_01001614	ice-bearing permafrost	sweet:realmSoil/Permafrost	Permafrost	skos:broadMatch	HumanCurated	"In SWEET this is a 3-D geospatial entity, instead of ENVO's environmental material."
+obo:ENVO_01001546	iceberg	sweet:realmCryo/Iceberg	Iceberg	skos:closeMatch	HumanCurated	"A candidate for exact match if SWEET adds ENVO's definition to SWEET's term"
+obo:ENVO_03000031	iceberg calving process	sweet:phenCryo/IceCalving	IceCalving	skos:broadMatch	HumanCurated	"SWEET lacks more specific terms under ice calving"
+obo:ENVO_01000882	icestorm	sweet:phenAtmoPrecipitation/IceStorm	IceStorm	skos:relatedMatch	HumanCurated	"In ENVO this is a process while in SWEET it is a type of Atmospheric precipitation which is a phenomenon (which is ambiguous). The relations between the two term's superclass is what leads us to say related instead of close."
+obo:ENVO_00000542	kame	sweet:realmLandGlacial/Kame	Kame	skos:relatedMatch	HumanCurated	"In SWEET a kame is a glacial landform - a type of geometrical object; in ENVO a kame is a kind of material entity."
+obo:ENVO_03400007	lake-effect snow storm	sweet:phenAtmoPrecipitation/LakeEffectSnowstorm	LakeEffectSnowstorm	skos:closeMatch	HumanCurated	"Candidate Exact match. SWEET needs a definition and depending on what that becomes this could become exact match"
+obo:ENVO_03400006	lake-effect snowfall	sweet:phenAtmoPrecipitation/LakeEffectSnow	LakeEffectSnow	skos:closeMatch	HumanCurated	"Candidate Exact match. In SWEET there is a long comment which should be turned into a definition.  SWEET talks about snow bands not the snow itself."
+obo:ENVO_03400008	lambing storm	sweet:phenAtmoPrecipitation/LambingStorm	LambingStorm	skos:closeMatch	HumanCurated	"Candidate Exact match. SWEET needs a definition and depending on what that becomes this could become exact match"
+obo:ENVO_01001547	land ice mass	sweet:realmCryo/LandIce	LandIce	skos:closeMatch	HumanCurated	"The SWEET definition is limited to the Earth while much of that text is a comment in ENVO and ENVO isn't scoped to the Earth"
+obo:ENVO_00000298	marine iceberg	sweet:realmCryo/Iceberg	Iceberg	skos:closeMatch	HumanCurated	"SWEET doesn't have this level of granularity"

```

Comment:
I don't understand the comment on granularity

Reply:
I think it just refers to the precision of the match where ENVO has the more specific term marine iceberg and sweet only iceberg.